mscore/pianoroll/pianolevelsfilter.h

```diff
@@ -54,7 +72,8 @@ class PianoLevelFilterOnTime : public PianoLevelsFilter {
       Q_DECLARE_TR_FUNCTIONS(PianoLevelFilterOnTime)
 
 public:
-      QString name() override { return tr("Note on time", "amount note 'on time' is adjusted by"); }
+      QString name() override { return tr(PianoLevelsFilterStrings[0]); }
```

Comment:
I don't think this works. `qApp->translate("PianoLevelsFilter", PianoLevelsFilterStrings[0])` should though

Reply:
And I'd do those in pianolevelschooser.cpp, here just the string